### PR TITLE
User curd

### PR DESCRIPTION
**Pull request by onifs10**

### CODE REVIEW

**Comment by onifs10** [comment]
would create a test case in the next PR

File: app/Repository/OrganizationUsers/OrganizationUserRepository.php [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    }
+
+
+    /**
+     * @inheritDoc
+     */
+    public function resetUserPassword($newPassword)
+    {
+        $organization_User = OrganizationUserModel::whereId($this->id)
+            ->update(['password' => bcrypt(trim($newPassword))]);
```
Comment (goodhands) [inline app/Repository/OrganizationUsers/OrganizationUserRepository.php:54]: Hmm why not use `Hash::make()`? That is the same method Laravel uses by default,

**Comment by onifs10** [comment]
Aiit... Would do that

File: app/Repository/OrganizationUsers/OrganizationUserRepository.php (hunk 1 above)
Comment (onifs10) [reply app/Repository/OrganizationUsers/OrganizationUserRepository.php:54]: Aiit .. I would do that

**Comment by onifs10** [comment]
@goodhands  i have added the Unit test

File: app/Http/Controllers/OrganizationUserController.php [hunk 2]
```diff
@@ -2,84 +2,86 @@
...
-    public function index()
+    public function passwordReset(Request $request, OrganizationUserInterface $model)
     {
-        //
-    }
+        try{
+            $response = $model->resetUserPassword(
+                $request->newPassword
+            );
+            if($response == false){
```
Comment (goodhands) [inline app/Http/Controllers/OrganizationUserController.php:36]: Falsy values are usually checked with a triple equal to sign (===)

Comment (onifs10) [reply app/Http/Controllers/OrganizationUserController.php:36]: changed

**Comment by goodhands** [comment]
One more thing you can push that later, I noticed you have a `POST` endpoint for changing displayName. Let's change that to `PATCH` or `PUT` instead

**Comment by onifs10** [comment]
aiit

**Comment by onifs10** [comment]
i would change logout to delete too

**Pull request merged**